lib/src/main/java/io/ably/lib/http/AsyncHttp.java

```diff
@@ -147,6 +148,8 @@ private AblyRequestWithFallback(
 		public void run() {
 			int retryCountRemaining = Hosts.isRestFallbackSupported(http.host) ? http.options.httpMaxRetryCount : 0;
 			String candidateHost = http.host;
+			if(http.options.fallbackHosts != null)
+				Collections.shuffle(http.options.fallbackHosts);
```

Comment:
No, sorry, we have to change it. I thought I had seen an earlier update that did that.